src/release_artifacts_resources/ios/cdk/cdk/credential_rotation/lambda_construct.py

```diff
@@ -43,3 +52,11 @@ def __init__(
         self.credential_rotator.add_environment(
             lambda_constants.GITHUB_PROJECT_PATH_ENV, "aws-amplify/aws-sdk-ios"
```

Comment:
We should consolidate this project path name along with the other hardcoded values (releases.amplify.aws; aws-sdk-ios-spm) into a single place to make it easier to refactor this when we have to do another rotator.

Reply:
We can take up as part of refactoring this rotator into a generic credential rotator.